### PR TITLE
feat(tracing): llama_index IO

### LINKED ISSUE #1077: [traces][llamaIndex] add semantic IO attributes to traces

### PR DESCRIPTION
**Pull request by mikeldking**
resolves #1077 

Adds input output values for llama_index traces

### CODE REVIEW

**Review by axiomofjoy — APPROVED** [review]

File: src/phoenix/trace/semantic_conventions.py [hunk 1]
```diff
@@ -53,8 +53,8 @@ class DeploymentAttributes(AttributeGroup):
 
 
 class MimeType(Enum):
-    TEXT = "text"
-    JSON = "json"
+    TEXT = "text/plain"
+    JSON = "application/json"
```
Comment (mikeldking) [inline src/phoenix/trace/semantic_conventions.py:57]: making these real MIME types: https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types

File: pyproject.toml [hunk 2]
```diff
@@ -97,15 +97,16 @@ dependencies = [
   "pytest-lazy-fixture",
   "arize",
   "langchain>=0.0.257",
-  "llama-index",
+  "llama-index>=0.8.1",
   "openai",
   "tenacity",
+  "nltk==3.8.1",
```
Comment (mikeldking) [inline pyproject.toml:103]: this gets imported by llama_index and I can't make it not fail the test. Gonna punt for now.

File: src/phoenix/experimental/callbacks/llama_index_trace_callback_handler.py [hunk 3]
```diff
@@ -74,14 +123,21 @@ def on_event_end(
...
             event_type=event_type,
             payload=payload,
             id_=event_id,
         )
 
+        # Parse the payload to extract the parameters
+        if payload is not None:
+            attributes = payload_to_semantic_attributes(payload)
+            for attribute, value in attributes.items():
+                event_data["attributes"][attribute] = value
```
Comment (RogerHYang) [inline src/phoenix/experimental/callbacks/llama_index_trace_callback_handler.py:139]: nit
```suggestion
            event_data["attributes"].update(payload_to_semantic_attributes(payload))
```

File: src/phoenix/experimental/callbacks/llama_index_trace_callback_handler.py [hunk 4]
```diff
@@ -58,13 +99,21 @@ def on_event_start(
...
             event_type=event_type,
             payload=payload,
             id_=event_id,
         )
+        event_data["attributes"] = {}
+        # Parse the payload to extract the parameters
+        if payload is not None:
+            attributes = payload_to_semantic_attributes(payload)
+            for attribute, value in attributes.items():
+                event_data["attributes"][attribute] = value
```
Comment (RogerHYang) [inline src/phoenix/experimental/callbacks/llama_index_trace_callback_handler.py:115]: nit
```suggestion
            event_data["attributes"].update(payload_to_semantic_attributes(payload))
```

**Review by RogerHYang — APPROVED** [review]

**Pull request merged**